sasl2-sys/build.rs

```diff
@@ -96,6 +96,16 @@ fn build_sasl(metadata: &Metadata) {
         .run()
         .expect("configure failed");
 
+    let make;
+    if metadata.host.contains("freebsd")
```

Comment:
dragonflybsd too. Also, given that we're giving up on BSD Make working, should we get rid of the BSD-specific logic on lines 119-123 ?

Reply:
All good points, thanks. Done.